_posts/2019/2019-07-24-how-to-transpile-node-modules-with-babel-and-webpack-in-a-monorepo.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: How to transpile node_modules with babel and webpack in a monorepo
+date: 2019-07-24
+header_image: public/webpack-babel-lerna.jpg
+header_position: center
+header_overlay: true
+category: coding
+tags: ["lerna", "babel", "webpack", "monorepo", "node_modules", "transpile"]
+authors: ["Somto"]
+about_authors: ["szikora"]
+---
+
+Recently, we faced an issue where Javascript stopped working on our storefront frontend on Internet Explorer 11.
+This came with the usual consequences such as `onClick` functions not not working etc.
```

Comment:
```suggestion
This came with the usual consequences such as `onClick` functions not working etc.
```

Reply:
Please don#tt forget me. I like your additions - let's hope Birgit doesn't find them too long.

Reply:
_Completely optional_
Just a comment from a rather linguistic point of view: I would either provide another example of a consequence before using `etc`, or strip `etc` altogether. 'etc' should, in my view, be generally used after a enumeration of at least 2 items.